compiler/noirc_evaluator/src/acir/mod.rs

```diff
@@ -1165,10 +1165,14 @@ impl<'a> Context<'a> {
                 ) {
                     // Subtractions must first have the integer modulus added before truncation can be
                     // applied. This is done in order to prevent underflow.
-                    let integer_modulus = power_of_two::<FieldElement>(bit_size);
-                    let integer_modulus = self.acir_context.add_constant(integer_modulus);
-                    var = self.acir_context.add_var(var, integer_modulus)?;
-                    max_bit_size += 1;
+                    // There is no underflow for field elements. Besides,
```

Comment:
Could we just expand this comment slightly? Something about how we expect the `max_bit_size` for a cast from a field to be `max_num_bits` and that is how we are gating this integer modulus addition?

Reply:
I updated the comment